src/Microsoft.TemplateEngine.Cli/TemplateUpdateChecker.cs

```diff
@@ -52,34 +83,11 @@ public async Task<IReadOnlyList<IUpdateUnitDescriptor>> CheckForUpdatesAsync(IRe
                 Reporter.Output.WriteLine(LocalizableStrings.UpdateCheckerNotAvailable.Bold().Red());
                 foreach (IInstallUnitDescriptor descriptor in descriptorsWithoutUpdaters)
                 {
-                    Reporter.Output.WriteLine($"\t{descriptor.UserReadableIdentifier}".Bold().Red());
+                    Reporter.Output.WriteLine($"\t{descriptor.UninstallString}".Bold().Red());
```

Comment:
We should make the spacing consistent with the other outputs, not sure how many other places we're using `\t` for output strings